week_10/진명재/week10_1043.java

```diff
@@ -0,0 +1,75 @@
+package week_10.진명재;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.*;
+import java.util.stream.IntStream;
+
+public class week10_1043 {
+    static ArrayList<Integer>[] arr;
+    public static void main(String[] args) throws Exception {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+
+        StringTokenizer st = new StringTokenizer(br.readLine());
+
+        int n = Integer.parseInt(st.nextToken());
+
+        int m = Integer.parseInt(st.nextToken());
+
+        int[] g = IntStream.rangeClosed(0, n + 1).toArray();
+        st = new StringTokenizer(br.readLine());
+        int k = Integer.parseInt(st.nextToken());
+        int[] truth = new int[k];
+        for (int i = 0; i < k; i++) {
+            truth[i] = Integer.parseInt(st.nextToken());
+        }
+
+        arr = new ArrayList[m];
+        for (int i = 0; i < m; i++) {
+            arr[i] = new ArrayList<>();
+            st = new StringTokenizer(br.readLine());
+            int p = Integer.parseInt(st.nextToken());
+            for (int j = 0; j < p; j++) {
+                arr[i].add(Integer.parseInt(st.nextToken()));
+            }
+        }
+
+
+
+        for (int i = 0; i < m; i++) {
+            int first_man = arr[i].get(0);
+            for (int j = 1; j < arr[i].size(); j++) {
+                group(first_man, arr[i].get(j),g);
+            }
+        }
+        int cnt = 0;
+        for (int i = 0; i < m; i++) {
+            int leader = arr[i].get(0);
+            boolean flag = true;
+            for (int j = 0; j < k; j++) {
+                if (find(leader,g) == find(truth[j],g)) {
+                    flag = false;
+                    break;
+                }
+            }
+            if (flag) {
+                cnt++;
+            }
+        }
```

Comment:
```suggestion
        Loop : for (int i = 0; i < m; i++) {
            int leader = arr[i].get(0);
            boolean flag = true;
            for (int j = 0; j < k; j++) {
                if (find(leader,g) == find(truth[j],g)) {
                    continue Loop;
                }
            }
            cnt++;
        }
```
반복문에 이름 붙이면 이렇게 처리 가능함